prusa/connect/printer/__init__.py

```diff
@@ -749,6 +752,12 @@ def loop_step(self):
                 elif res.status_code == 202 and item.timeout > time():
                     self.queue.put(item)
                     sleep(1)
+            elif isinstance(item, CameraRegister):
+                if res.status_code == 202 and item.timeout > time():
+                    self.queue.put(item)
+                    sleep(1)
+                else:
+                    self.camera_mgr.registration_cb(res)
```

Comment:
Is there cammera identificator in response? Resp. token is unique for each camera, so camera manager needs camera id too.